scripts/sql/ai_schema_compare.py

```
@@ -0,0 +1,217 @@
+  prompt = PROMPT_TMPL.format(sql_a=sql_a, sql_b=sql_b)
+
+  gen_cfg = GenerationConfig(
+      temperature=0.0,
+      top_p=0.1,
+      top_k=32,
+      max_output_tokens=1024,
+      response_mime_type="application/json",
+  )
+
+  last_err = None
+  for attempt in range(3):
```

Ruff (0.12.2)

Loop control variable `attempt` not used within loop body

Rename unused `attempt` to `_attempt`

(B007)